### PR TITLE
Articles hub

### PR DESCRIPTION
**Pull request by wendisha**
# Description

Rebuild Articles Hub in React

Fixes # (issue)

https://trello.com/c/i9i3TnvV/34-make-updates-to-article-hub

### CODE REVIEW

File: resources/js/articles/ArticlesListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ArticlesListItem = ({ article }) => {
+    const baseUrl = window.location.href;
+    return (
+        <a href={`${baseUrl}/${article.slug}`} className="flex flex-wrap mb-6">
+            <div className="md:w-2/12 mb-2 md:mb-0">
+                <img src={article.image} alt="Article Illustration." />
```
Comment (sbsmith86) [inline resources/js/articles/ArticlesListItem.jsx]: for the alt value , we should do something more descriptive so screen readers will understand the difference between each article item. `alt={article.title}` should work.

File: resources/js/articles/ArticlesLayout.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const ArticlesLayout = () => {
+    return (
+        <div className="max-w-body pt-12 pb-8 xl:px-0">
+            <h1 className="font-bold leading-tight text-5xl">Articles To Explore</h1>
+            <p>
+                <a href="https://www.lantern.co/register" className="text-fauna underline">
```
Comment (sbsmith86) [inline resources/js/articles/ArticlesLayout.jsx]: can you update this link to be a relative link so `<a href="/register" />`

This way on different environments we won't always point to a production url, we will go to the register page relative to the environment the user is on (i.e http://mvp.test, http://mvp-staging.herokuapp.com)

File: resources/js/articles/ArticlesListItem.jsx (hunk 1 above)
Comment (wendisha) [reply resources/js/articles/ArticlesListItem.jsx]: Done.

File: resources/js/articles/ArticlesLayout.jsx (hunk 2 above)
Comment (wendisha) [reply resources/js/articles/ArticlesLayout.jsx]: Done.

**Pull request merged**